### DISCUSSION TITLE
`null` vs `undefined`

### DISCUSSION THREAD

**Discussion by mmoskal** [microsoft/devicescript#35] [General]
ES has two "none" values `null` and `undefined`. Should we unify them?

There is little runtime overhead either way, but we have to decide.

**Comment by mmoskal** [comment]
We stick to ES semantics.

**Reply by pelikhan** [reply]
@mmoskal can you start a page in the docs documentating all these subtle differences?

**Reply by mmoskal** [reply]
for reference: https://github.com/microsoft/devicescript/blob/main/website/docs/Language/ds-vs-js.md
